frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -0,0 +1,307 @@
+<template lang='pug'>
+table.table.table-in-card.c-payments(:class='{"c-payments-edit": paymentsType === "edit"}')
+  thead
+    tr
+      th(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(
+          @change='checkAllpayment'
+          type='checkbox'
+        )
+      th {{ titles.one }}
+      th.c-payments-amount {{ titles.two }}
+      th.c-payments-date {{ titles.three }}
+
+  tbody
+    tr(
+      v-for='(payment, index) in payments'
+      :key='index'
+    )
+      td(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(type='checkbox' v-model='payment.checked')
+      td
+        .c-user
+          avatar-user.c-avatar(:username='payment.to' size='xs')
+          strong.c-name {{payment.to}}
+        .c-user-month(:class='index === 0 ? "has-text-1" : "c-status-danger c-status"') {{ moment(payment.date).format('MMMM') }}
+      td.c-payments-amount(v-if='paymentsType !== "edit"')
+        .c-payments-amount-info(v-if='index === 0 && (!needsIncome && paymentsType === "todo")')
+          i18n.c-payments-amount-text.has-text-1(
+            tag='span'
+            :args='{partial_amount: `<strong class="has-text-0">$20</strong>`, partial_total: currency(payment.amount)}'
+          ) {partial_amount} out of {partial_total}
+
+          i18n.c-status.c-status-primary Partial
+          //- i18n.c-status.c-status-primary(
+          //-   tag='span'
+          //-   role='alert'
+          //-   v-if='payment.hash'
+          //-   :class='payment.paymentClass'
+          //- ) {{ payment.paymentStatusText }}
+
+        strong(v-else) {{currency(payment.amount)}}
+        .c-payments-amount-info(
+          v-if='index === 0 && (needsIncome || paymentsType === "received")'
+          :class='"c-status-warning"'
+        )
+          tooltip.c-name-tooltip(
+            v-if=''
+            direction='top'
+            :isTextCenter='true'
+            :text='L("{personName} marked this payment as not received.", { personName: payment.to })'
+          )
+            .button.is-icon-smaller.c-tip
+              i.icon-info
+
+          i18n.c-status Not received
+
+      td
+        .c-actions
+          .c-actions-month(:class='!(index !== 0 && paymentsType === "todo") ? "has-text-1" : "c-status-danger c-status"') {{ moment(payment.date).format('MMMM') }}
+          payments-list-menu(
+            v-if='paymentsType !== "edit"'
+            :paymentsType='paymentsType'
```

Comment:
When paymentsType is "received" the menu is empty. I think it should show "Payment details" and "I did not receive this"